tests/api-suite-info/00-get-graph-raw-1.t

```diff
@@ -43,7 +43,7 @@ cmp_ok "${SUITE_RUN_DIR}/ctb-get-graph-raw.out" <<'__OUT__'
         ], 
         [
             "t1.1", 
-            null, 
+            "T.1", 
```

Comment:
The old logic was returning a list by using `.values()` method of a dict. The new logic sorts the list to fix the ordering.